release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -0,0 +1,77 @@
+## 2022-07-05 Version 2.1.0 Release Notes
```

Comment:
Curious how this list is generated ? 
Is it manual today?

Reply:
I was following the steps mentioned by @peternied [here](https://github.com/opensearch-project/security/pull/1772) to just generate the list of all commit. And yes we still have to manually go through each PR and filter out it into either of the sections (like Enhancements/bug fixes/Maintainance)

Reply:
@Rishikesh1159 Glad to see this documentation was useful.  What do you think about amending your commit message to include these steps as well, like from [2b92ca9](https://github.com/opensearch-project/security/pull/1772/commits/2b92ca91ad910fddbb8481d9db9d3e2e136a9950)?

Reply:
Yeah sure that's a good idea. Let me amend my commit